clients/cli/src/system.rs

```diff
@@ -35,50 +32,11 @@ pub fn process_memory_gb() -> f64 {
     memory as f64 / 1000.0 / 1000.0 / 1000.0 // Convert to GB
 }
 
-/// Estimate peak FLOPS (in GFLOP/s) of this machine based on the number of cores and clock speed.
-#[allow(unused)]
-pub fn estimate_peak_gflops() -> f32 {
-    // Estimate peak FLOPS based on the number of cores and a rough estimate of operations per cycle
-    // Assuming 4 operations per cycle (e.g., add, multiply, divide, sin)
-    let num_cores = num_cores() as f32;
-    let peak_flops = num_cores * 4.0 * 2.0e9; // TODO: Assuming 2 GHz clock speed
-    (peak_flops / 1e9) as f32 // Convert to GFLOP/s
-}
-
-/// Estimate FLOPS (in GFLOP/s) of this machine.
-pub fn measure_gflops() -> f32 {
-    const NUM_TESTS: u64 = 1_000_000;
-    const OPERATIONS_PER_ITERATION: u64 = 4; // sin, add, multiply, divide
-    const NUM_REPEATS: usize = 5; // Number of repeats to average the results
-
-    let num_cores: u64 = match available_parallelism() {
-        Ok(cores) => cores.get() as u64,
-        Err(_) => {
-            eprintln!("Warning: Unable to determine the number of logical cores. Defaulting to 1.");
-            1
-        }
-    };
-    let avg_flops: f64 = (0..NUM_REPEATS)
-        .map(|_| {
-            let start = Instant::now();
-
-            let total_flops: u64 = (0..num_cores)
-                .into_par_iter()
-                .map(|_| {
-                    let mut x: f64 = 1.0;
-                    for _ in 0..NUM_TESTS {
-                        x = black_box((x.sin() + 1.0) * 0.5 / 1.1);
-                    }
-                    NUM_TESTS * OPERATIONS_PER_ITERATION
-                })
-                .sum();
-
-            total_flops as f64 / start.elapsed().as_secs_f64()
-        })
-        .sum::<f64>()
-        / NUM_REPEATS as f64; // Average the FLOPS over all repeats
-
-    (avg_flops / 1e9) as f32
+/// Estimate peak FLOPS (in GFLOP/s) from the number of prover threads and clock speed.
+pub fn estimate_peak_gflops(num_provers: usize) -> f32 {
+    // Assuming 4 operations per cycle
+    let peak_flops = (num_provers as f32) * 4.0 * 2.0e9; // Assumes 2 GHz clock speed
```

Comment:
Is there no way to actually get the clock speed? Even if the other test was inaccurate it was at least derived from concrete measurements.